services/src/main/java/org/fao/geonet/api/records/MetadataValidateApi.java

```diff
@@ -136,6 +153,26 @@ Reports validateRecord(
 
         Locale locale = languageUtils.parseAcceptLanguage(request.getLocales());
 
+        boolean isSubtemplate = metadata.getDataInfo().getType() == MetadataType.SUB_TEMPLATE;
+        boolean validSet = (isvalid != null);
+        if (!isSubtemplate && validSet) {
+            throw new BadParameterEx("isvalid", "<not Null for not subTemplate>");
```

Comment:
I don't undertstand, I tried something like
"throw new BadParameterEx(String.format("Parameter isValid (%s) should be defined if and only if validating subtemplate."), isvalid)"
but i got MissingFormatArgumentException
It' s true that with one "%s" String.format expect at least one extract parameter,
Wonder how it works in 
https://github.com/geonetwork/core-geonetwork/blob/develop/services/src/main/java/org/fao/geonet/api/records/MetadataSocialApi.java#L132